toolbox/incari/object/get-position.md

```diff
@@ -0,0 +1,31 @@
+# Overview
+
+![The Get Position Node.](../../../.gitbook/assets/toolbox/incari/object/get-position.PNG)
+
+| Input | Output |
+| :--- | :--- |
+| Object ID | Position |
+
+**Get Position** returns the *position* of an **Object** in a **Scene**. The **Object ID** is given as input to the **Node**.
+
+# Inputs
+
+|Input|Type|Description|
+|---|---|---|
+|*Pulse Input* (►)|**Pulse**|A standard **Input Pulse**, to trigger the execution of the **Node**.|
+|`Object ID`|Int|The ID of the **Object** whose *position* is to be determined.
```

Comment:
Object ID is a data type itself, not an int. Please see the comment below regarding Vector3 and the snippets in VSCode.